acme_nginx/Acme.py

```diff
@@ -60,6 +62,23 @@ def __init__(
         self.chain = "https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem"
         self.dns_provider = dns_provider
         self.skip_nginx_reload = skip_nginx_reload
+        self.update_date_threshold_days = update_date_threshold_days
+        
+        self.IsOutOfDate = True
+        if self.update_date_threshold_days:
+            try:
+                certTime = datetime.fromtimestamp(os.path.getmtime(self.cert_path))
```

Comment:
This method is quite simple but not reliable, would be better to get date from certificate itself.

Reply:
Hi, it's not really the same thing because here, certTime is the date of the last update from acme-nginx.
But that can be replace by the date of the emission of the certificate (not the expiration date)

Reply:
True, that can be date of the certificate emission.